model-server/src/test/kotlin/org/modelix/model/server/handlers/RepositoriesManagerTest.kt

```diff
@@ -26,6 +26,16 @@ class RepositoriesManagerTest {
         store.close()
     }
 
+    @Test
+    fun `deleted branches are not listed anymore`() = runTest {
```

Comment:
Not sure if the test case name really reflects what is actually tested. Isn't this more in the direction of "deleting the last branch works"?

Reply:
renamed